checks/macos.py

```diff
@@ -6,7 +6,7 @@
 
 
 def getMacVersionLine(lines):
-    isMac = search('OS Name: Mac OS X', lines)
+    isMac = search('OS Name: Mac OS X', lines) + search('OS Name: macOS', lines)
```

Comment:
Does `search` return `0` when no matches are found?

Reply:
IIRC, `search` returns `[]` when no matches are found.

Reply:
Ok, in theory this can work, because `[] + []` is falsy, so `if isMac:` should resolve to `False` as well.

I'd prefer this to be more explicit, but I guess it's fine for the given purpose.

Reply:
isMac, despite the name, is not expected to be a boolean.

Reply:
Yep, this is consistent across the analyzer.